tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/UploadAdditionalDocumentSD.java

```diff
@@ -0,0 +1,92 @@
+package ca.bc.gov.open.jag.efiling.stepDefinitions;
+
+import ca.bc.gov.open.jag.efiling.helpers.SubmissionHelper;
+import ca.bc.gov.open.jag.efiling.models.UserIdentity;
+import ca.bc.gov.open.jag.efiling.services.OauthService;
+import ca.bc.gov.open.jag.efiling.services.SubmissionService;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import io.restassured.specification.MultiPartSpecification;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Assert;
+import org.springframework.core.io.ClassPathResource;
+
+import java.io.File;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.UUID;
+
+public class UploadAdditionalDocumentSD {
+
+    private final OauthService oauthService;
+    private final SubmissionService submissionService;
+
+    private static final String TEST_DOCUMENT_PDF = "test-document.pdf";
+    private static String DOCUMENTS_PATH = "documents";
+
+    private UUID actualTransactionId;
+    private Response actualDocumentResponse;
+    private String actualSubmissionId;
+    private UserIdentity actualUserIdentity;
+    private Response actualAdditionalDocumentResponse;
+
+    public Logger logger = LogManager.getLogger(UploadAdditionalDocumentSD.class);
+
+    public UploadAdditionalDocumentSD(OauthService oauthService, SubmissionService submissionService) {
+        this.oauthService = oauthService;
+        this.submissionService = submissionService;
+        actualTransactionId = UUID.randomUUID();
+    }
+
+    @Given("valid admin user credentials are authenticated")
+    public void userAccountIsAuthenticated() {
+
+        actualUserIdentity = oauthService.getUserIdentity();
+    }
+
+    @When("request is made to upload additional documents")
+    public void userSubmitAdditionalPdfDocument() throws IOException {
+        logger.info("Submitting request with submit parameters");
+
+        File resource = new ClassPathResource(
+                "data/test-document.pdf").getFile();

```

Comment:
Not a big issue. MessageFormat.format("data/{0}", TEST_DOCUMENT_PDF). Also if you are finding you are using that reference frequently you could create a Keys class to put all of your re-used constants.

Reply:
Awesome. Thanks @TayGov!! fixed on other classes as well.